python/bindings/include/openravepy/openravepy_environmentbase.h

```diff
@@ -292,9 +292,9 @@ class PyEnvironmentBase : public OPENRAVE_ENABLE_SHARED_FROM_THIS<PyEnvironmentB
     void SetUserData(object o);
     object GetUserData() const;
 
-    void SetUnit(std::string unitname, dReal unitmult);
+    py::dict GetUnitInfo() const;
```

Comment:
You can't just kill the old `SetUnit`, too many places use it.

Reply:
You can always set it as deprecated so that c++ generates a compile-time warning.

Reply:
Sure. I add them back.